index.html

```diff
@@ -19,20 +19,19 @@
 
 	<body>
 		<div class="reveal">
-                  <div class="slides">
+    <div class="slides">
 		    <section id="titleSlide">
 			<h2> Modern Programming for Data Analytics - Class Slides </h2>
 		    </section>
 		    <section>
-			    <h3>Syed Zia Uddin</h3>
+			    <h3>Nikhil Kashyap</h3>
 			    <ul>
-		              <li>Gouripur, a small town in Bangladesh
-			      <li>Your major
-		              <li>Shawshank's Redemption
-		  	     </ul>
+		        <li>Bangalore, India
+			      <li>Masters in Information Systems and Managament
+		        <li>Coming to America- Eddie Murphy
+		  	  </ul>
 		    </section>
 		    <section>
-				copy and paste yours
```

Comment:
You did not follow the instructions, I asked you to copy the slide to here, please conform to that instruction.